Makefile

```diff
@@ -1,33 +1,128 @@
+TARGETS     := darwin/amd64 darwin/arm64 linux/amd64 linux/386 linux/arm linux/arm64 linux/ppc64le linux/s390x windows/amd64
+TARGET_OBJS ?= darwin-amd64.tar.gz darwin-amd64.tar.gz.sha256 darwin-amd64.tar.gz.sha256sum darwin-arm64.tar.gz darwin-arm64.tar.gz.sha256 darwin-arm64.tar.gz.sha256sum linux-amd64.tar.gz linux-amd64.tar.gz.sha256 linux-amd64.tar.gz.sha256sum linux-386.tar.gz linux-386.tar.gz.sha256 linux-386.tar.gz.sha256sum linux-arm.tar.gz linux-arm.tar.gz.sha256 linux-arm.tar.gz.sha256sum linux-arm64.tar.gz linux-arm64.tar.gz.sha256 linux-arm64.tar.gz.sha256sum linux-ppc64le.tar.gz linux-ppc64le.tar.gz.sha256 linux-ppc64le.tar.gz.sha256sum linux-s390x.tar.gz linux-s390x.tar.gz.sha256 linux-s390x.tar.gz.sha256sum windows-amd64.zip windows-amd64.zip.sha256 windows-amd64.zip.sha256sum
+BINNAME     ?= iter8
+ITER8_IMG ?= iter8/iter8:latest
+
+GOBIN         = $(shell go env GOBIN)
+ifeq ($(GOBIN),)
+GOBIN         = $(shell go env GOPATH)/bin
+endif
+GOX           = $(GOBIN)/gox
+
+# go options
+TAGS       :=
+LDFLAGS    := -w -s
+GOFLAGS    :=
+
+# Rebuild the binary if any of these files change
+SRC := $(shell find . -type f -name '*.go' -print) go.mod go.sum
+
+# Required for globs to work correctly
+SHELL      = /usr/bin/env bash
+
+GIT_TAG    = $(shell git describe --tags --abbrev=0 --exact-match 2>/dev/null)
+
+ifdef VERSION
+	BINARY_VERSION = $(VERSION)
+endif
+BINARY_VERSION ?= ${GIT_TAG}
+
+# Only set Version if building a tag or VERSION is set
+ifneq ($(BINARY_VERSION),)
+	LDFLAGS += -X github.com/iter8-tools/iter8/basecli/RootCmd.Version=${BINARY_VERSION}
+endif
+
+LDFLAGS += $(EXT_LDFLAGS)
+
+.PHONY: all
+all: build
+
+# ------------------------------------------------------------------------------
+#  build
+
+.PHONY: build
+build: $(BINDIR)/$(BINNAME)
+
+$(BINDIR)/$(BINNAME): $(SRC)
+	GO111MODULE=on go build $(GOFLAGS) -trimpath -tags '$(TAGS)' -ldflags '$(LDFLAGS)' -o '$(BINDIR)'/$(BINNAME) ./
+
+.PHONY: cmddocs
+cmddocs:
+	go run cmd/docs/main.go	
+
+# ------------------------------------------------------------------------------
+#  install
+
+.PHONY: install
+install: build
+	@install "$(BINDIR)/$(BINNAME)" "$(INSTALL_PATH)/$(BINNAME)"
+
+# ------------------------------------------------------------------------------
+#  dependencies
+
+# If go get is run from inside the project directory it will add the dependencies
+# to the go.mod file. To avoid that we change to a directory without a go.mod file
+# when downloading the following dependencies
+
+$(GOX):
+	(cd /; GO111MODULE=on go get -u github.com/mitchellh/gox)
+
+# ------------------------------------------------------------------------------
+#  release
+
+.PHONY: build-cross
+build-cross: LDFLAGS += -extldflags "-static"
+build-cross: $(GOX)
+	GOFLAGS="-trimpath" GO111MODULE=on CGO_ENABLED=0 $(GOX) -parallel=3 -output="_dist/{{.OS}}-{{.Arch}}/$(BINNAME)" -osarch='$(TARGETS)' $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' ./
+
+.PHONY: dist
+dist:
```

Comment:
should be dependent on build-cross; otherwise get error since `_dist` may not exist.

Reply:
Fixed.